lib/trino-filesystem-azure/src/test/java/io/trino/filesystem/azure/AbstractTestAzureFileSystem.java

```diff
@@ -185,7 +186,7 @@ protected final Location getRootLocation()
     @Override
     protected final void verifyFileSystemIsEmpty()
     {
-        assertThat(blobContainerClient.listBlobs()).isEmpty();
+        assertThat(blobContainerClient.listBlobs()).map(BlobItem::getName).isEmpty();
```

Comment:
this change feels strange to me. i don't see its value.

Reply:
see the commit message